frontend/src/pages/[lang]/cfp/is-cfp-open.graphql

```diff
@@ -2,5 +2,9 @@ query IsCfpOpen($conference: String!) {
   conference(code: $conference) {
     id
     isCFPOpen
+    deadlines {
```

Comment:
you can fetch a single deadline: https://github.com/pythonitalia/pycon/blob/5d0cac9834203f96204e423eb0b797c9704c041a/frontend/src/pages/index.graphql#L29

Reply:
Oh yeah, of course, I didn't see it!